<launch>

    <!-- <node pkg="frontier_exploration" type="explore_client" name="explore_client" output="screen"/>

    <node pkg="frontier_exploration" type="explore_server" name="explore_server" output="screen">

    <param name="frequency" value="1.0"/>

    <param name="goal_aliasing" value="2.0"/>

    <rosparam file="$(find kobuki_navigation)/param/costmap_common_params.yaml" command="load" ns="explore_costmap" />
    <rosparam file="$(find kobuki_navigation)/param/costmap_exploration.yaml" command="load" ns="explore_costmap" />

  </node> -->


    <node name="explore" pkg="explore_lite" type="explore" output="screen">
        <param name="cmd_vel_topic" value="/cmd_vel"/>
        <param name="map_topic" value="/map"/>
        <param name="goal_topic" value="/move_base/goal"/>
        <param name="explore_costmap_topic" value="/move_base/global_costmap/costmap"/>
    </node>


</launch>
